src/QGCFileDownload.cc

```diff
@@ -76,6 +76,7 @@ bool QGCFileDownload::download(const QString& remoteFile, bool redirect)
     networkRequest.setAttribute(QNetworkRequest::User, localFile);
 
     QNetworkReply* networkReply = get(networkRequest);
+    networkReply->ignoreSslErrors();
```

Comment:
I think @dogmaphobic deals with this sort of things in map tiles differently. Would be good to be consistent with that. @dogmaphobic ?